src/main/kotlin/study/racingCar/domain/RacingCarLogic.kt

```diff
@@ -0,0 +1,16 @@
+package study.racingCar.domain
+
+class RacingCarLogic {
+    fun startRacing(carNum: Int, movingTryNum: Int): List<MutableList<Int>> {
+        val cars = (0 until carNum).map { Car() }
+        val movedStepsRecords = (0 until movingTryNum).map { (0 until carNum).map { 0 }.toMutableList() }
+
+        for (i in 0 until movingTryNum) {
+            for (j in 0 until carNum) {
+                cars[j].moveOneStep()
+                movedStepsRecords[i][j] = cars[j].movedSteps
+            }
+        }
+        return movedStepsRecords
+    }
```

Comment:
`return`이 `MutableList`가 있어서 조금 아쉽군요!  아래의 코드를 참고해볼까요?
핵심은 `map{}`에 있습니다
```suggestion
    fun startRacing(carNum: Int, movingTryNum: Int): List<List<Int>> {
        val cars = (0 until carNum).map { Car() }

        return (0 until movingTryNum).map {
            cars.map { car ->
                car.moveOneStep()

                car.movedSteps
            }
        }
    }
```